connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -98,6 +98,52 @@ abstract class AvroSuite
     }, new GenericDatumReader[Any]()).getSchema.toString(false)
   }
 
+  /* Check whether an Avro schema of union type is converted to SQL in an expected way, when the
```

Comment:
Is this supposed to be Javadoc? If so, then it should look like this: 
```
/**
 * <your text>
 */
```

If not, you can just use the inline comments.